.gitignore

```diff
@@ -1 +1,7 @@
-node_modules
+/node*
+*.env*
+.eslintrc.js
+.editorconfig
```

Comment:
Doesn't look like there's a `.editorconfig` in here btw, prob not a massive deal though?